cfn/collector.template.template

```diff
@@ -0,0 +1,779 @@
+                        "logs:CreateLogStream",
+                        "logs:PutLogEvents"
+                     ],
+                     "Resource":[ {
+                       "Fn::Join":["",
+                          [
+                             "arn:aws:logs:", { "Ref":"AWS::Region" },
+                             ":", { "Ref":"AWS::AccountId" },
+                             ":log-group:/aws/lambda/", { "Ref":"CollectLambdaFunction" },
+                             ":log-stream:*"
+                          ]
+                       ]
+                     }]
+                  },
+                  {
+                     "Effect":"Allow",
+                     "Action":[
+                        "lambda:*"
+                     ],
+                     "Resource":[{
+                       "Fn::GetAtt":[
+                           "CollectLambdaFunction",
+                           "Arn"
+                       ]
+                     }]
+                  },
+                  {
+                     "Effect":"Allow",
+                     "Action":[
+                        "s3:Get*"
+                     ],
+                     "Resource":[{
+                       "Fn::Join":["",
+                          [
+                             "arn:aws:s3:::",
+                             {
+                                "Fn::Join" : ["", [
+                                    { "Ref" : "PackagesBucketPrefix" }, "-",
+                                    { "Ref" : "AWS::Region" }
+                                ]]
+                             },
+                             "/*"
+                          ]
+                       ]
+                     }]
+                  },
+                  {
+                     "Effect":"Allow",
+                     "Action":[
+                        "sqs:*"
+                     ],
+                     "Resource":[{
+                       "Fn::GetAtt":[
+                           "PawsPollStateQueue",
+                           "Arn"
+                       ]
+                     }]
+                  }
+               ]
+            }
+         }
+      },
+      "HealthCheckLambdaPolicy":{
+         "Type":"AWS::IAM::Policy",
+         "DependsOn":[
+              "CollectLambdaRole"
+         ],
+         "Properties":{
+            "Roles":[
+               {
+                  "Ref":"CollectLambdaRole"
+               }
+            ],
+            "PolicyName":"alertlogic-health-check-lambda-policy",
+            "PolicyDocument":{
+               "Version":"2012-10-17",
+               "Statement":[
+                  {
+                     "Effect":"Allow",
+                     "Action":[
+                        "cloudformation:DescribeStacks"
+                     ],
+                     "Resource":[{
+                         "Fn::Join":[ "",
+                              [
+                                  "arn:aws:cloudformation:", { "Ref":"AWS::Region" },
```

Comment:
Please take the latest version of okta temaplate. I think this bug was fixed there. Basically the `Resource` should be `*` here.